src/OrbitVulkanLayer/SubmissionTracker.h

```diff
@@ -293,6 +317,16 @@ class SubmissionTracker : public VulkanLayerProducer::CaptureStatusListener {
     absl::WriterMutexLock lock(&mutex_);
     bool marker_depth_exceeds_maximum;
     {
```

Comment:
We can actually remove this scope, I think, as we moved the lock to the full scope anyway. Would help with the nestedness.